website/docs/docs/cloud/use-visual-editor.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Use the visual editor" 
+id: use-visual-editor      
+sidebar_label: "Use the visual editor" 
+description: "Access and use the visual editor to create or edit dbt models through a visual, drag-and-drop experience inside of dbt Cloud." 
+---
+
+# Use the visual editor <Lifecycle status='beta'/> 
+
+:::tip Beta feature
+The visual editor provides users with a seamless and drag-and-drop experience inside of dbt Cloud. It's available in private beta for [dbt Cloud Enterprise accounts](https://www.getdbt.com/pricing). 
+
+To join the private beta, [register your interest](https://docs.google.com/forms/d/e/1FAIpQLScPjRGyrtgfmdY919Pf3kgqI5E95xxPXz-8JoVruw-L9jVtxg/viewform) or reach out to your account team to begin this process.
+:::
+
```

Comment:
i think our best practice here is to start the page with at least a short paragraph vs a callout. i recall getting review comments from matt and/or leona that a page that starts with a callout negatively impacts our SEO. i forget the details atm tho. would be good to check-up on it, i think

Reply:
i think you can have both - if the content is relevant (e.g. enterprise and beta), has keywords, and helpful to the user, the SEO impact is probably minimal. especially if the search engine's understanding of the page content is clear. 

however, this has inspired me to do something I've been wanting to try for a while and i think i can meet you halfway here and add introductory text after the title and then the callout, satisfying both our points:
![Screenshot 2024-06-05 at 12 00 26](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/06928375-0e9a-470b-aa7b-f1823e503b45)

Reply:
this is super duper cool!